File: Day 3/Class Work/Employee.java
package com.coforge.day3;

public class Employee {
	private int eid;
	private String ename;
	private double esalary;
	
	public Employee(int eid, String ename, double esalary) {
		this.eid = eid;
		this.ename = ename;
		this.esalary = esalary;
	}
	
	public void setSalary(double esalary) {
		this.esalary = esalary;
	}
	public double getSalary() {
		return esalary;
	}
	
	public void display() {
		System.out.println("Employee Id: "+eid);
		System.out.println("Employee Name: "+ename);
		System.out.println("Employee Salary: "+esalary);
	}
}

File: Day 26/Assignment/Employee.js
import { Component } from "react";

class Employee extends Component{
    render(){
        return(
            <div className="card">

                <div className="card-header bg-primary text-white">
                    <h3>Employee Details</h3>
                </div>

                <div className="card-body">
                    <p><strong>Employee ID : </strong>{this.props.eid}</p>
                    <p><strong>Employee Name : </strong>{this.props.ename}</p>
                    <p><strong>Department : </strong>{this.props.department}</p>
                    <p><strong>Designation : </strong>{this.props.designation}</p>
                    <p><strong>Salary : </strong>{this.props.salary}</p>
                </div>
            </div>

        );
    }
}
export default Employee;